test/unit/server/BaseHttpServerFactory.test.ts

```diff
@@ -125,4 +125,75 @@ describe('A BaseHttpServerFactory', (): void => {
       expect(res.text).toBe(`${error.stack}\n`);
     });
   });
+
+  describe('A Base HttpServerFactory (With Unix Sockets)', (): void => {
+    const socket = getSocket('BaseHttpServerFactory');
+    const httpOptions = {
+      http: true,
+      showStackTrace: true,
+    };
+
+    afterAll(async(): Promise<void> => {
+      server.close();
+    });
+
+    beforeEach(async(): Promise<void> => {
+      jest.clearAllMocks();
+    });
+
+    if (!(process.platform === 'win32')) {
```

Comment:
```suggestion
    if (process.platform !== 'win32') {
```

Also alternative suggestion to this structure: have describe blocks here nested in the bigger describe block, one for unix and one for windows. And at the top of those describe blocks have something like:
```ts
if (process.platform === 'win32') {
  return;
}
```
Which would then prevent the entire describe block being executed in that case, and similarly for the other block.